packages/zowe-explorer/src/utils/LoggerUtils.ts

```diff
@@ -29,12 +29,22 @@ export class ZoweLogger {
     public static zeOutputChannel: vscode.OutputChannel;
     private static defaultLogLevel: "INFO";
     private static zeLogLevel: string;
+    private static displayName: string;
+    private static zeVersion: string;
 
     public static async initializeZoweLogger(context: vscode.ExtensionContext): Promise<void> {
         try {
             const logsPath: string = ZoweVsCodeExtension.customLoggingPath ?? context.extensionPath;
             globals.initLogger(logsPath);
-            await this.initVscLogger(context, logsPath);
+            await this.initVscLogger(logsPath);
```

Comment:
Should we call `initVscLogger` after displayName and zeVersion have been defined?

Reply:
Yes, thanks for spotting this - addressed in f40a368